examples/todo/components/TodoItem.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import { Todo } from '../types'
+import { useSelector, atom } from 'klyva'
+import { RemovableAtom } from 'klyva/dist/types'
+import { TodoItemView } from './TodoItemView'
+import { TodoItemForm } from './TodoItemForm'
+
+type TodoItemProps = {
+  todoAtom: RemovableAtom<Todo, any>
```

Comment:
You can use `RemovablePrimitiveAtom` here so, or in place of `any` put `SetState<Todo> `